src/app/globals.css

```
@@ -3,18 +3,242 @@
+    box-shadow: 0 0 0 2px var(--accent);
+  }
+}
+
+/* Custom slider styles - More specific to override Tailwind */
+input[type="range"].slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  background: transparent;
+  height: 6px;
+  border-radius: 3px;
+  background: var(--border);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)